internal/registration/api/inprocess/interface.go

```diff
@@ -0,0 +1,29 @@
+package inprocess
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+)
+
+type RegisterNewUserCommand struct {
+	FirstName string `json:"firstName"`
```

Comment:
remove `json` struct tags from this struct. All consumers of this API for now should map it manualy.